show/main.py

```diff
@@ -201,6 +201,8 @@ def run_command(command, display_cmd=False):
     proc = subprocess.Popen(command, shell=True, stdout=subprocess.PIPE)
 
     while True:
+        if return_cmd:
+            return proc.stdout.read()
```

Comment:
This will not work if output has a lot of data.
You need to use communicate() for that. https://docs.python.org/2/library/subprocess.html#subprocess.Popen.communicate
Also
```
Warning Use communicate() rather than .stdin.write, .stdout.read or .stderr.read to avoid deadlocks due to any of the other OS pipe buffers filling up and blocking the child process.
```
from https://docs.python.org/2/library/subprocess.html

Reply:
Thanks Pavel. Using communicate() from next iteration